### ISSUE TITLE
Editing board title doesn't work on web Cboard

### ISSUE THREAD

**Issue by mzilak** [cboard-org/cboard#710]
After creating a new board while logged in, I wanted to change its title on home screen but after clicking on Accept, it's not saved but returned to previous title.

![image](https://user-images.githubusercontent.com/57595384/84698760-a80ead00-af50-11ea-9175-751d669580b6.png)

**Comment by mzilak** [comment]
After refreshing the page, the new title occurs.

**Comment by sylvansson** [comment]
I'd be happy to work on this @martinbedouret.

**Comment by sylvansson** [comment]
@mzilak I wasn't able to reproduce this issue. Do you get this behaviour consistently? If so, could you share your browser version and the exact steps to reproduce?

Thanks in advance.

**Comment by mzilak** [comment]
@sylvansson , unfortunately, I did not manage to reproduce this again. This happened that one time while I was preparing one board so it is not consistent. If I get this behavior again, I will let you know.
Also. while I was trying to reproduce this bug, I noticed that while the user is not logged in, the title of the board/folder cannot be changed. Is this the expected behavior (on both web and mobile version)?
